easybuild/easyconfigs/l/LLVM/LLVM-14.0.3-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,40 @@
+name = 'LLVM'
+version = '14.0.3'
+
+homepage = "https://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'cstd': 'gnu++11', 'pic': True}
+
+source_urls = ['https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s/']
+sources = ['llvm-%(version)s.src.tar.xz']
```

Comment:
add source urls `https://github.com/numba/llvmlite/raw/v0.41.1/conda-recipes/` and the 3 patches;
```python3
patches = [
    'llvm14-clear-gotoffsetmap.patch',
    'llvm14-remove-use-of-clonefile.patch',
    'llvm14-svml.patch'
]
```